test/integration/scenarios/network/peer_disconnect.js

```diff
@@ -119,18 +119,27 @@ module.exports = params => {
 			});
 
 			describe('node stop and start', () => {
-				it('stop all the nodes in the network', done => {
```

Comment:
Where does the restart of all of the nodes take place now?

Reply:
Here all the nodes will be restarted except "node_0" https://github.com/LiskHQ/lisk/pull/2071/files#diff-2be26c6f881769b542eb6b22cd0aabe7L131